### PR TITLE
Implement ReadMe history entity

### PR DESCRIPTION
**Pull request by xtuchyna**

### CODE REVIEW

File: srcopsmetrics/entities/readme_history.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from github.ContentFile import ContentFile as GithubContentFile
+from voluptuous.schema_builder import Schema
+from datetime import datetime
+
+from srcopsmetrics.entities import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ReadMeHistory(Entity):
```
Comment (pacospace) [inline srcopsmetrics/entities/readme_history.py:34]: shouldn't be `ReadMeHistory(ReadMe)`?

File: srcopsmetrics/entities/readme_history.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    """GitHub ReadMe entity."""
```
Comment (pacospace) [inline srcopsmetrics/entities/readme_history.py:35]: `README History entity`?

File: srcopsmetrics/entities/readme_history.py (hunk 1 above)
Comment (xtuchyna) [reply srcopsmetrics/entities/readme_history.py:34]: oh, ofc! thxx francesco

**Comment by xtuchyna** [comment]
Not relevant now, therefore closing this PR.

**Pull request closed without merging**